### ISSUE TITLE
Installation problem with setup.py

### ISSUE THREAD

**Issue by vijayvarma392** [sxs-collaboration/gwsurrogate#44]
From @oshaughn.

Question re gwsurrogate maintenance: I'm hitting an error like [this one](https://github.com/man-group/pytest-plugins/issues/87) in [this autobuild](https://git.ligo.org/james-clark/research-projects-RIT/-/jobs/2909634) where pip install of the release gwsurrogate fails due to 

```
#8 2.508 long\_description = pypandoc.convert('README.md', 'rst')
#8 2.508 AttributeError: module 'pypandoc' has no attribute 'convert'
```
which seems to rely on deprecated use of a convert routine in the setup file.  Is this something that could be fixed upstream?

**Comment by vijayvarma392** [comment]
@sfield17 or @duetosymmetry: ideas?

**Comment by duetosymmetry** [comment]
Looks like @raffienficiaud tried to address this in #43? But the description there did not mention this change.

In https://github.com/JessicaTegner/pypandoc/commit/2cfcc1e32ad23986e2edbfa799619b0691f9ddd5, `pypandoc.convert` became unavailable.

**Comment by raffienficiaud** [comment]
@duetosymmetry I have not checked thoroughly the generation of the documentation... but I believe this piece of code is not needed anymore as Markdown is supported natively by `pypi`, see eg. https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/

**Comment by sfield17** [comment]
Fixed with PR #45
